Tasks/AzurePowerShellV5/AzurePowerShell.ps1

```diff
@@ -14,6 +14,10 @@ $input_pwsh = Get-VstsInput -Name pwsh -AsBool -Default $false
 $input_workingDirectory = Get-VstsInput -Name workingDirectory -Require
 $validateScriptSignature = Get-VstsInput -Name validateScriptSignature -AsBool
 
+$hostEnv = "ADO/AzurePowerShell@v5`_$($env:AGENT_NAME)`_$($env:BUILD_BUILDID)`_$($env:RELEASE_RELEASEID)`_$($env:BUILD_DEFINITIONNAME)`_$($env:RELEASE_DEFINITIONNAME)`_$($env:AGENT_OS)"
```

Comment:
Here and in *.ts fiel - let's use the following order of items:
AGENT_OS, AGENT_NAME, BUILD_DEFINITIONNAME, BUILD_BUILDID, RELEASE_DEFINITIONNAME, RELEASE_RELEASEID